From 067cb2bae391310dbb0b224179913ca8accddd58 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Fri, 21 Feb 2014 16:10:31 -0500
Subject: [PATCH 1/2] FEDORA: Soften requires.txt

We don't need hard requirements on the Django version or dateutil
because RPM handles these gracefully.
---
 setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index 8a35e13b5c3269d9033f79f4d2bb2ee5f377c10f..35c50920d90a49ac3050460738f416101825895e 100755
--- a/setup.py
+++ b/setup.py
@@ -171,16 +171,16 @@ setup(name=PACKAGE_NAME,
       },
       cmdclass=cmdclasses,
       install_requires=[
-          'Django>=1.4.10,<1.5',
+          'Django>=1.4,<1.5',
           'django_evolution>=0.6.9,<0.7',
           'Djblets>=0.7.29,<0.8',
           'django-pipeline>=1.2.24,<1.3',
           'docutils',
           markdown_requirement,
-          'mimeparse>=0.1.3',
+          'python_mimeparse>=0.1.3',
           'paramiko>=1.9.0',
           'Pygments>=1.5',
-          'python-dateutil==1.5',
+          'python-dateutil<2.0',
           'python-memcached',
           'pytz',
           'recaptcha-client',
-- 
1.9.0

